editor/editor_node.cpp

```diff
@@ -2775,6 +2775,16 @@ void EditorNode::_editor_select(int p_which) {
 	}
 }
 
+void EditorNode::select_editor_by_name(const String &p_name) {
+	if ("" == p_name)
```

Comment:
We write conditions the other way around in general, `p_name == ""`.

Should it print an error if no editor is found that matches `p_name`?